### PR TITLE
Add description check to PRs

### PR DESCRIPTION
**Pull request by mattstratton**
In theory, this should provide a check to see if pull requests have removed the default text, and it should fail the check if they did not. 

I have now removed the offending text in this description! the PR should now show green.

adding the word Bluth to trigger to check the new message!

### CODE REVIEW

**Comment by mattstratton** [comment]
for what it's worth, the check ran (and passed) on the first submission of this PR
image

I'll try to edit the PR to put the default text back (which should trigger the test again) and it should fail it.

**Comment by mattstratton** [comment]
Cool! when the default text is in the PR, it fails a check:

image

image

**Comment by mattstratton** [comment]
ookay I like this now because it also makes the comment and tags the submitter!

File: .github/workflows/pr.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+            echo "PR description does not have default text removed"
+            exit 1
+          fi
+      - name: Comment on PR
+        id: comment
+        if: failure() && steps.prcheck.outcome == 'failure'
+        uses: thollander/actions-comment-pull-request@v2
+        with:
+          message: |
+            Hi @${{ github.event.pull_request.user.login }}! Please remove the default text from the pull request description!
```
Comment (phrawzty) [inline .github/workflows/pr.yml:32]: The functionality is great, though maybe this message doesn't drive the behaviour we want. Maybe something like?
```suggestion
            Hi @${{ github.event.pull_request.user.login }}! It looks like you've left the default text in the pull request description. Please replace that text with something meaningful. Thanks!

```

Comment (phrawzty) [inline .github/workflows/pr.yml:32]: The functionality is great, though maybe this message doesn't drive the behaviour we want. Maybe something like?
```suggestion
            Hi @${{ github.event.pull_request.user.login }}! It looks like you've left the default text in the pull request description. Please replace that text with something meaningful. Thanks!

```

**Review by phrawzty — CHANGES_REQUESTED** [review]

File: .github/workflows/pr.yml (hunk 1 above)
Comment (mattstratton) [reply .github/workflows/pr.yml:32]: yeah this is better. I'll accept it and test!

**Comment by mattstratton** [comment]
> Hi @mattstratton! It looks like you've left the default text in the pull request description. Please replace that text with something meaningful. Thanks!

@phrawzty tada!

**Pull request merged**